### PR TITLE
Upstream v2.60.1

### PR DESCRIPTION
**Pull request by mininny**
Sync upstream v2.60.1 https://github.com/ledgerwatch/erigon/releases/tag/2.60.1

### CODE REVIEW

File: turbo/jsonrpc/eth_call.go [hunk 1]
```diff
@@ -154,56 +154,50 @@ func (api *APIImpl) EstimateGas(ctx context.Context, argsOrNil *ethapi2.CallArgs
...
 	if err != nil {
 		return 0, err
 	}
-	chainConfig, err := api.chainConfig(ctx, dbtx)
-	if err != nil {
-		return 0, fmt.Errorf("read chain config: %v", err)
-	}
-	if chainConfig.IsOptimismPreBedrock(blockNum) {
+
+	if chainConfig.IsOptimismPreBedrock(latestCanBlockNumber) {
```
Comment (mininny) [inline turbo/jsonrpc/eth_call.go:167]: Upstream change of getting the block number from dbtx

**Comment by ImTei** [comment]
@mininny 👍 please resolve the CI failure

**Comment by mininny** [comment]
> @mininny 👍 please resolve the CI failure

The tests are fixed! :) https://github.com/testinprod-io/op-erigon/pull/184/commits/bb122856418339ff92571224d9e1e5addeabf8cb

**Review by ImTei — APPROVED** [review]

**Comment by mininny** [comment]
Added the op-erigon version bump up to 0.6.1: https://github.com/testinprod-io/op-erigon/pull/184/commits/ebeb01d20ba144b2b128e58bf0dcb0d4590ed4a1

**Pull request merged**